File: Tareas/Unidad2/a2253332351_practica04/39_ParImpar.cpp
#include <iostream>
using namespace std;

int main() {
    int numero, residuo;

    cout << "Introduce un numero entero: ";
    cin >> numero;

    residuo = numero % 2;

    if (residuo == 0) {
        cout << numero << " es PAR" << endl;
    } else {
        cout << numero << " es IMPAR" << endl;
    }

    return 0;
}

File: Tareas/Unidad2/a2253332351_practica04/39_ParImpar.java
import java.util.Scanner;

public class ParImpar {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        int numero, residuo;

        System.out.print("Introduce un numero entero: ");
        numero = entrada.nextInt();

        residuo = numero % 2;

        if (residuo == 0) {
            System.out.println(numero + " es PAR");
        } else {
            System.out.println(numero + " es IMPAR");
        }
    }
}
